### PR TITLE
Fix not testing packages in directories

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
`duckpan test` was not testing anything within a directory.

### CODE REVIEW

File: lib/App/DuckPAN/Cmd/Test.pm [hunk 1]
```diff
@@ -23,7 +23,7 @@ sub run {
 		$self->app->emit_error("Could not find dist.ini.") unless -e "dist.ini";
 		$self->app->emit_error("Could not begin testing. Is Dist::Zilla installed?") if $ret = system("dzil test");
 	} else {
-		$self->app->emit_error("Tests failed! See output above for details") if $ret = system("prove -Ilib");
+		$self->app->emit_error("Tests failed! See output above for details") if $ret = system("prove -Ilib t/*");
```
Comment (zachthompson) [inline lib/App/DuckPAN/Cmd/Test.pm]: I think we want:
``` Perl
system 'prove -lr t';
```
here to catch arbitrarily nested subdirs.

Comment (GuiltyDolphin) [inline lib/App/DuckPAN/Cmd/Test.pm]: @zachthompson You're right.

I was thinking of adding tests to make sure directories work (and for regression) - I can't see any tests that are testing the existing `duckpan test`? Any pointers?

Comment (zachthompson) [inline lib/App/DuckPAN/Cmd/Test.pm]: `prove -r` should take care of directories but if you think of other tests that might be valuable let us know.

**Comment by zachthompson** [comment]
Taken care of in https://github.com/duckduckgo/p5-app-duckpan/pull/307

**Pull request closed without merging**